### PR TITLE
Upgrade React Native

### PR DESCRIPTION
**Pull request by TheTreek**
# Summary of Changes

Updated React Native to 0.64.1 and enabled hermes

# Related Issues

https://github.com/hyperledger/indy-sdk/issues/2346

# Pull Request Checklist

This is just a reminder about the most common mistakes. Please make sure that you tick all _appropriate_ boxes. But please read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!

If an item doesn't apply to your pull request, **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this).
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components.
- [x] Run prettier: `npm run style-format`
- [x] Updated **documentation** for changed code and new or modified features.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -25,6 +25,10 @@ const App = (props) => {
   let location = useLocation()
   let history = useHistory()
 
+  useEffect(()=>{
+    console.log("Hermes enabled:", !!global.HermesInternal)
```
Comment (manghuli) [inline App.js]: do we keep these kind of console.logs in?

**Review by manghuli — DISMISSED** [review]

File: App.js (hunk 1 above)
Comment (JamesKEbert) [reply App.js]: No, we should remove this log IMO

**Review by JamesKEbert — CHANGES_REQUESTED** [review]
@NeilSMyers could you fix the merge conflict with the podfile lock? Removing that log would probably also be helpful.

**Review by JamesKEbert — APPROVED** [review]
This is great. Thanks for your efforts here @TheTreek and @NeilSMyers

**Pull request merged**